### DISCUSSION TITLE
Translation completed! <Portuguese (Portugal)>

### DISCUSSION THREAD

**Discussion by cong862023** [sircharlo/meeting-media-manager#2033] [Translations]
### What language do you speak?

Hello!
Recently, I started the translation work for this language (TPO/pt-pt). 
I am pleased to announce that today I have finished the process of translation and review for the language: Portuguese (Portugal) (TPO/pt-pt). I hope my work has been useful. 
I would be very happy if the complete translation is included in one of the next updates of the M³. 
Greetings from Portugal to everyone involved in the development of this fantastic app.

**Comment by mtdvlpr** [comment]
@cong862023, thank you very much! It will be included in the next release, and it's already included in the latest beta release, which can be received by enabling beta updates in the settings.

**Reply by sircharlo** [reply]
Yes, thanks so much @cong862023 !

**Comment by cong862023** [comment]
👍👍
